Doc/library/asyncio-policy.rst

```diff
@@ -40,6 +40,10 @@ for the current process:
 
    Return the current process-wide policy.
 
+   .. deprecated:: next
+      The :func:`get_event_loop_policy` function is deprecated and
+      will be removed in Python 3.16.
```

Comment:
You should also add an entry to the `Doc/deprecations/pending-removal-in-3.16.rst`

Reply:
I have planned to write more docs at end.